Xteam.sh

```diff
@@ -27,8 +27,8 @@ echo " "
 echo -e " $Red       ||----------------------------$Cyan [features] $Blue ---------------------------||"
 echo -e " $Red       ||                                                                    ||"
 echo -e " $Red       ||             $Purple%=>$Yellow[1️⃣] Insta information gathering$Blue                     ||"
-echo -e " $Red       ||             $Purple%=>️$Yellow[2️⃣] Crack android lockscreen interfaces$Blue             ||"
-echo -e " $Red       ||             $Purple%=>$Yellow[3️⃣] Phishing hacks$Blue                                  ||"
+echo  -e  " $ Red        ||              $ Purple ٪ => ️ $ Yellow [2️⃣] كسر واجهات قفل شاشة android $ Blue2              ||"
```

Comment:
Sorry. Wouldn't it be better if the `||"` delimiter was parallel to the previous line? It might look small but it would be neater.